src/screens/Account/EmptyStateAccount.js

```diff
@@ -81,9 +81,9 @@ class EmptyStateAccount extends PureComponent<{
                 <LText semiBold color="darkBlue">
                   {account &&
                     account.currency &&
-                    listTokenTypesForCryptoCurrency(mainAccount.currency).join(
-                      ", ",
-                    )}
+                    listTokenTypesForCryptoCurrency(mainAccount.currency)
+                      .map(c => c?.toUpperCase() ?? c)
```

Comment:
https://github.com/LedgerHQ/ledgerjs/blob/8138e3edd573997c8c403df3241dcd95937cc81e/packages/cryptoassets/src/tokens.ts#L66

it can't be falsy so you can do `c => c.toUpperCase()`